edx_when/admin.py

```diff
@@ -0,0 +1,63 @@
+"""
+Django admin support for edx-when.
+"""
+
+from django.contrib import admin
+
+from .models import ContentDate, DatePolicy, UserDate
+
+
+@admin.register(ContentDate)
+class ContentDateAdmin(admin.ModelAdmin):
+    list_display = [
+        'course_id',
+        'location',
+        'field',
+        'policy',
+        'active',
+    ]
+    list_filter = ['active']
+    search_fields = ['course_id', 'location']
+    ordering = ['course_id', 'policy']
+
+    raw_id_fields = ['policy']  # dropdown of dates is not helpful
+
+    # Specified just to control ordering
+    fields = ['course_id', 'location', 'field', 'policy', 'active']
+
+
+@admin.register(DatePolicy)
+class DatePolicyAdmin(admin.ModelAdmin):
+    search_fields = ['abs_date', 'rel_date']
+
+
+@admin.register(UserDate)
+class UserDateAdmin(admin.ModelAdmin):
+    list_display = [
+        'user',
+        'course_id',
+        'location',
+        'field',
+        'date',
+    ]
+    search_fields = ['user__username', 'content_date__course_id', 'content_date__location']
+    ordering = ['user', 'content_date__course_id', 'content_date__policy']
+
+    raw_id_fields = ['content_date', 'user']
+    exclude = ['actor']
+
+    def course_id(self, obj):
+        return obj.content_date.course_id
+
+    def location(self, obj):
+        return obj.content_date.location
+
+    def field(self, obj):
+        return obj.content_date.field
+
+    def date(self, obj):
+        return obj.abs_date or obj.rel_date
+
+    def save_model(self, request, obj, form, change):
+        obj.actor = request.user
+        super().save_model(request, obj, form, change)
```

Comment:
Is this appropriate for the `actor` field @cpennington, or should I not try to be clever and just leave `actor` in place when editing via the admin form?

Reply:
I'm not sure where that `actor` field is coming from. Is that from the history model?

Reply:
No, it's a top level field defined specifically in `UserDate` -- I assume it's fillled from the user that publishes in Studio?

Reply:
Er, not publishes, but overrides the user schedule rather. Same person that fills in `reason` I guess.

Reply:
Ah, ok. Yeah, I'd record the current admin user, then.